pkg/xds/bootstrap/generator_test.go

```diff
@@ -292,10 +294,11 @@ var _ = Describe("bootstrapGenerator", func() {
 
 		// given
 		params := bootstrap_config.DefaultBootstrapParamsConfig()
-		params.XdsHost = "127.0.0.1"
+		params.XdsHost = "localhost"
```

Comment:
Please note that `localhost` does not resolve always to `127.0.0.1`. For example:

```
$ uname -a
Darwin ............ 19.6.0 Darwin Kernel Version 19.6.0: Thu Oct 29 22:56:45 PDT 2020; root:xnu-6153.141.2.2~1/RELEASE_X86_64 x86_64

$  curl -vv localhost:80
*   Trying ::1...
* TCP_NODELAY set
* Connection failed
* connect to ::1 port 80 failed: Connection refused
*   Trying 127.0.0.1...
* TCP_NODELAY set
* Connection failed
* connect to 127.0.0.1 port 80 failed: Connection refused
* Failed to connect to localhost port 80: Connection refused
* Closing connection 0
curl: (7) Failed to connect to localhost port 80: Connection refused
```

Reply:
Yup, that's true. The cert we were using for tests had `localhost` not `127.0.0.1` therefore I thought it was easier to change it here. We are not resolving localhost to `127.0.0.1` in this test.